distribution/docs/src/main/resources/_contents/_endpoints/metrics-endpoint-contents.adoc

```diff
@@ -0,0 +1,221 @@
+==== Metrics Endpoint
+
+[NOTE]
+====
+EXPERIMENTAL
+====
+
+[WARNING]
+====
+Note that the Metrics endpoint URL says "internal."
```

Comment:
Is this our standard blurb for internal things? If not, we may want to rephrase it a little. The last part _"any custom applications built to make use of it, as described below, should be made with caution"_ sounds a little odd.